index.js

```diff
@@ -50,12 +50,9 @@ exports.decorateTerms = (Terms, { React }) => {
         );
         if (canvasCollection.length >= 3) {
           const canvases = new Array();
-          document.body.removeEventListener(
-            'DOMSubtreeModified',
-            canvasListener,
-          );
+
           for (let canvas of canvasCollection) canvases.push(canvas);
-          this.setState({ ...this.state, canvases });
+          this.setState({ canvases });
```

Comment:
The spread operator is used correctly in the original code assuming ES6 standards (which I believe production version of Hyper is on ES5). Object.assign() is a fallback for the spread operator.
Merging your suggestion, Josh, with the original:
```suggestion
          this.setState((prevState, _) => Object.assign({}, prevState, {canvases}));
```